Source/UnitTests/GTMSessionFetcherTestServer.m

```diff
@@ -220,6 +220,15 @@ - (NSURL *)localURLForFile:(NSString *)name {
   return [NSURL URLWithString:urlString];
 }
 
+- (NSURL *)localURLForFileUsingAppend:(NSString *)name {
+  // We need to create http URLs referring to the desired
+  // resource to be found by the http server running locally.
+  // Return a localhost:port URL for the test file
+
+  NSString *urlString = [NSString stringWithFormat:@"http://localhost:%d", _server.port];
+  return [[NSURL URLWithString:urlString] URLByAppendingPathComponent:name];
```

Comment:
Isn't this the same results as calling the existing helper?

Reply:
When the tests are run using `swift test` command, trying to initialize an NSURL with a string that is not percent encoded will fail and the result will be nil.